debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleOffsetContext.java

```diff
@@ -184,9 +185,9 @@ public static Builder create() {
             }
             else {
                 final Scn scn = sourceInfo.getScn();
-                final Scn commitScn = sourceInfo.getCommitScn();
+                final String commitScn = sourceInfo.getCommitScn().toString();
```

Comment:
@Naros Please don't use toString but a structural approach. So mutliple keys with a common prefix like we do in `TransactionContext`